recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,147 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._exceptions import RecipeScrapersExceptions
+from ._utils import get_minutes, normalize_string
+
+
+class UnsupportedLocale(RecipeScrapersExceptions):
+    """No support for detected recipe locale on this webpage"""
+
+    def __init__(self, lang, url):
+        self.lang = lang
+        self.url = url
+        message = f'Detected recipe locale "{self.lang}" is not supported for url={url}'
+        super().__init__(message)
```

Comment:
Two small adjustments for this new exception class:

  * Please move the class definition into the `recipe_scrapers._exceptions` module.
  * Please also add it to the `__all__` list declared in the `recipe_scrapers.__init__` module.